tests/run-make/rustc-crates-on-stable/rmake.rs

```diff
@@ -0,0 +1,42 @@
+//! Checks if selected rustc crates can be compiled on the stable channel (or a "simulation" of it).
+//! These crates are designed to be used by downstream users.
+
+use run_make_support::{cargo, run_in_tmpdir, rustc_path, source_root};
+
+fn main() {
+    run_in_tmpdir(|| {
+        // Use the stage0 beta cargo for the compilation (it shouldn't really matter which cargo we
+        // use)
+        let cargo = cargo()
```

Comment:
The variable is unused, let's remove `let cargo = `.

Reply:
done